util/reggen/uvm_reg_base.sv.tpl

```diff
@@ -57,19 +57,19 @@ ${make_ral_pkg_hdr(dv_base_prefix, [])}
 ${make_ral_pkg_fwd_decls(esc_if_name, rb.type_regs, rb.windows)}
 % for r in rb.all_regs:
 <%
+    mr = null
```

Comment:
This is Python, where the null object is called `None` :-)

Reply:
python should give me an error for this. Thanks for pointing it out. Fixed